README.md

```diff
@@ -49,6 +49,21 @@ $.ajax({
     });
 ```
 
+## Allow default to all ajax requests
```

Comment:
Please update this to `Add retry logic for all ajax requests`

Reply:
Fixed